README.md

```diff
@@ -326,6 +326,56 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+<a name="multi-condition-case-when"></a>
+* put multiple when conditions on separate lines
```

Comment:
According to other rules in the guide: 
1. The sentence must be capitalized and end with a dot.
2. Good and bad examples usually combined into single code snippet.